test/parallel/test-util-promisify.js

```diff
@@ -143,7 +143,7 @@ const stat = promisify(fs.stat);
 
   o.fn = fn;
 
-  o.fn().then(common.mustCall(function(val) {
+  o.fn().then(common.mustCall((val) => {
```

Comment:
Hey, very nice, thank you! :tada: 

I'm ok as it is now, but what do you think about this way?

```suggestion
  o.fn().then(common.mustCall((val) => assert(val)));
```

Reply:
Thank you, It looks more readable 👍

Reply:
Thanks!

Would you like to make that change?